module-4/backend/week_two.md

```diff
@@ -1,22 +1,55 @@
 ## Week Two - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What is Webpack and why is it useful?
+  *  Webpack is a node package that helps with the process of shipping code to production. Code that is optimized for a developer is not the most efficient for the browser. For this reason, Webpack helps with the transition from development code to production code. Webpack looks at our starter file and runs all the dependencies throughout our project. It will take care of preprocessers (i.e. SCSS) and minify our code into a single file for the browser to read.
+
+
 2. When do you want to use event delegation?
+  *  We want to use event delegation to avoid adding event listeners to multiple children. This can cause problems such as memory leak. Instead we want to put an event listener on a single parent.
```

Comment:
yup, this also helps when you want to bind an event to an element that is dynamically loaded or not on the page on page load 👍